src/clang-cpp-frontend/clang_cpp_convert.cpp

```diff
@@ -1016,18 +1023,25 @@ bool clang_cpp_convertert::name_param_and_continue(
    *     (done as part of the clang_c_converter's get_function_param flow)
    */
 
-  assert(id.empty() && name.empty());
-
   const clang::DeclContext *dcxt = pd.getParentFunctionOrMethod();
-  if(is_cpyctor(dcxt) && is_defaulted_ctor(dcxt))
+  if(const auto *md = llvm::dyn_cast<clang::CXXMethodDecl>(dcxt))
   {
-    // let's deal with the unnamed parameter for an implicit defaulted cpyctor
-    get_cpyctor_name(
```

Comment:
This function has been merged with name_param_and_continue. Now it becomes unused, we should remove it.